config/sync/user.role.os2loop_user_administrator.yml

```diff
@@ -116,6 +118,7 @@ permissions:
   - 'delete own os2loop_question content'
   - 'delete own os2loop_section_page content'
   - 'delete revisions'
+  - 'delete terms in os2loop_media_library'
```

Comment:
This looks like stuff from https://github.com/itk-dev/os2loop/pull/161 and should be removed from here.